pkg/aaparser/aaparser.go

```diff
@@ -26,7 +26,7 @@ func GetVersion() (int, error) {
 // LoadProfile runs `apparmor_parser -r` on a specified apparmor profile to
 // replace the profile.
 func LoadProfile(profilePath string) error {
-	_, err := cmd("-r", filepath.Dir(profilePath))
+	_, err := cmd("", "-r", filepath.Dir(profilePath))
```

Comment:
Maybe this should be `-a`? I'm not sure from reading the man page what `apparmor_parser "" -r /tmp/blah` does.